sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,154 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+class CoulombKineticFriction(ForceActuator):
+    """Coulomb kinetic friction with Stribeck and viscous effects.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+    .. math::
+        F = f_c * sign(v) + (f_{max} - f_c) * exp(-(v / v_s)**2) + sigma * v
```

Comment:
```suggestion
        F = f_c sign(v) + (f_{max} - f_c)  exp(-(v / v_s)**2) + sigma  v
```

`*` is for code, not math.

Reply:
Also exp() is code not math.

Reply:
I updated to
```
Explanation
===========

This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
described by the function:

.. math::
    F = f_c sign(v) + (f_{max} - f_c) e^{-(v / v_s)^2} + sigma v

where :math:`f_c` is the Coulomb friction constant, :math:`f_{max}` is the maximum static friction force,
:math:`v_s` is the coefficient of sliding friction, :math:`sigma` is the viscous friction constant,
and :math:`v` is the relative velocity.

The default friction force is :math:`F = mu_k N`, where :math:`N` is the normal force.
When specified, the actuator includes:

- Stribeck effect: :math:`(mu_s - mu_k) N e^{-(v / v_s)^2}`
- Viscous effect: :math:`sigma v`

In case we include all effects, the full actuator represents:

.. math::
    F = mu_k N sign(v) + (mu_s - mu_k) N e^{-(v / v_s)^2} + sigma v
```